Source/WebCore/platform/Length.cpp

```diff
@@ -357,6 +357,27 @@ Length convertTo100PercentMinusLength(const Length& length)
     return makeCalculated(CalcOperator::Subtract, Length(100, LengthType::Percent), length);
 }
 
+Length convertTo100PercentMinusLengthSum(const Length& a, const Length& b)
+{
+    // FIXME: The main simplification code does not deal with substract expressions so this does some basic steps.
+    // Blink has a seperate calc node type for pixel-and-percent values so they don't need to be expressed as sums,
```

Comment:
Why this comment? Do we care how Chromium is different?

Reply:
It was to point out a possible solution.